### DISCUSSION TITLE
Using Math

### DISCUSSION THREAD

**Discussion by kishan042** [six7/figma-tokens#405] [Q&A] [answered]
Hello! 

I'm trying to use maths to multiple an alias `{spacing.small} * 2` where a token called "small" is 8px and another token called medium should use the token small * 2. However, when I apply the medium token it seems to only apply the value of the alias. 

Can someone point out what I'm doing wrong?

![image](https://user-images.githubusercontent.com/15107911/149789799-89c8cfe6-5634-4c4f-8da2-686a20b1db01.png)

**Comment by six7** [comment, marked as answer]
Remove the `px` from the `spacing.small` token for it to work. You generally don't need to add units as the plugin will default to `px`.

**Reply by kishan042** [reply]
Thank you so much!
